R/cal-estimate-logistic.R

```diff
@@ -136,7 +144,10 @@ cal_logistic_impl <- function(.data,
       source_class = source_class
     )
   } else {
-    stop_multiclass()
+    msg <- paste("The number of outcome factor levels isn't consistent with",
+                 "the calibration method. The levels were:",
+                 paste0("'", levels, "'", collapse = ", "))
```

Comment:
The error should mention what the expected number of levels should be

```suggestion
    msg <- paste("The number of outcome factor levels isn't consistent with",
                 "the calibration method. Only two class `truth` factors are",
                 "allowed. The given levels were:",
                 paste0("'", levels, "'", collapse = ", "))
```